### DISCUSSION TITLE
Firmware Sonoff original

### DISCUSSION THREAD

**Discussion by Pante90** [indomus/forum#1259] [Hardware & Firmware]
Goodmorning, 
i want to ask if someone have the original firmware sonoff basic r3, because i flashed a esp8285 with tasmota, and now sonoff doesn't turn on.  Meanwhile, i want to try to put over the basic r3, a firmware of TH16 for create th16 with basic r3. For now i ask if someone have a original firmware of sonoff basic r3. 
Thanks a lot

**Comment by KC-inDomus** [comment]
got it. write me on dm

**Reply by Pante90** [reply]
i can't write you on dm, because i'm new on forum and i don't found the button. 
Can you send me the file on my email? abonomi90@gmail.com
Thanks a lot.
